app/src/main/java/co/touchlab/kmmbridgekickstartandroid/MainApp.kt

```diff
@@ -15,15 +12,13 @@ class MainApp : Application() {
     override fun onCreate() {
         super.onCreate()
 
-        initAnalytics(AndroidAnalytics())
-        AppAnalytics.appStarted()
+        val sdkHandle = startSDK(analytics = AndroidAnalytics(), context = this)
```

Comment:
Shouldn't the SDK handle be a member of the class, rather than a local variable?